### PR TITLE
Appconfig readmes

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/appconfiguration/azure-appconfiguration/samples/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+products:
+  - azure
+  - azure-app-configuration
+---
+
+# Azure App Configuration Data Library Python Samples
+
+## Prerequisites
+
+You must have an [Azure subscription][azure_sub], and a [Configuration Store][configuration_store] to use this package.
```
Comment (kaerm) [inline sdk/appconfiguration/azure-appconfiguration/samples/README.md]: the links not linked?

File: sdk/appconfiguration/azure-appconfiguration/samples/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+To create a Configuration Store, you can use the Azure Portal or [Azure CLI][azure_cli].
```
Comment (kaerm) [inline sdk/appconfiguration/azure-appconfiguration/samples/README.md]: same here about links

File: sdk/appconfiguration/azure-appconfiguration/samples/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+After that, create the Configuration Store:
```
Comment (kaerm) [inline sdk/appconfiguration/azure-appconfiguration/samples/README.md]: I would combine this with the previous line to make sure people do not think there is a next step:

To create a Configuration Store, you can either use Azure Portal[link to portal] or if you are using [Azure CLI][azure_cli] you can simply run the following snippet in your console: 
`that cli line you had there`

File: sdk/appconfiguration/azure-appconfiguration/samples/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+## Setup
+
+Install the Azure App Configuration client library for Python with pip:
+
+```commandline
+pip install azure-appconfiguration
+```
+
```
Comment (kaerm) [inline sdk/appconfiguration/azure-appconfiguration/samples/README.md]: is there no dependency on identity?

Comment (xiangyan99) [reply sdk/appconfiguration/azure-appconfiguration/samples/README.md]: No. azure-core does not depend on identity. We will have identity dependencies after we have AAD support.

File: sdk/appconfiguration/azure-appconfiguration/README.md [hunk 5]
```diff
@@ -23,7 +23,7 @@ Modern programs, especially programs running in a cloud, generally have many com
 
 Use the client library for App Configuration to create and manage application configuration settings.
 
-[Source code]() | [Package (Pypi)][package] | [API reference documentation]() | [Product documentation][azconfig_docs]
+[Source code]() | [Package (Pypi)][package] | [API reference documentation][appconfig_rest] | [Product documentation][azconfig_docs]
```
Comment (kaerm) [inline sdk/appconfiguration/azure-appconfiguration/README.md]: here the link should be to SDK ref docs, not rest

File: sdk/appconfiguration/azure-appconfiguration/samples/README.md (hunk 4 above)
Comment (kaerm) [reply sdk/appconfiguration/azure-appconfiguration/samples/README.md]: I meant in the samples :)

Comment (xiangyan99) [reply sdk/appconfiguration/azure-appconfiguration/samples/README.md]: Right. Samples do not depend on identity either. :)

File: sdk/appconfiguration/azure-appconfiguration/README.md [hunk 6]
```diff
@@ -1,11 +1,29 @@
+- [Azure App Configuration client library for Python](#azure-app-configuration-client-library-for-python "Azure App Configuration client library for Python")
```
Comment (kaerm) [inline sdk/appconfiguration/azure-appconfiguration/README.md]: Also, I completely missed this - why did you decide to move this section to the top? I don't think we need a menu. I'd just keep the examples (lines 11-14 here) listed (if anything at all) at the beginning of the examples section.

**Pull request merged**